src/App.css

```diff
@@ -1,3 +1,60 @@
 .App {
+  font-family: 'Roboto', sans-serif;
```

Comment:
Is this the prototype font or the final, either way is totally fine.

Reply:
I was rolling with this font at first because material-ui requires it; however, I would like to go with something nicer. Do you have any fonts that you really like?

Reply:
lol not at all. I am to fonts as you are to colors